lib/banner.js

```diff
@@ -19,21 +19,22 @@ module.exports = (type, heading, messages = []) => {
     }
   }
 
-  const formatedMessages = messages
-    .map((message) => identString(wrap(message, 80), 4))
-    .join('\n\n');
+  const gutter = 4;
+  const fullWidth = process.stdout.columns < 80 ? process.stdout.columns : 80;
+  const contentWidth = fullWidth - gutter * 2;
+
+  const border = highlight([...Array(fullWidth)].map(() => '-').join(''));
```

Comment:
Heads up using `fill` would be much easier here:
```suggestion
  const border = highlight(Array(fullWidth).fill('-').join(''));
```

But totally not a blocker.